bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -178,24 +178,42 @@ export class PlotCanvasView extends BokehView
     # Update any DataRange1ds here
     frame = @model.frame
     bounds = {}
+    log_bounds = {}
+
+    calculate_log_bounds = false
+    for r in _.values(frame.x_ranges).concat(_.values(frame.y_ranges))
+      if r instanceof DataRange1d
+        if r._mapper_type == "log"
```

Comment:
I'd be OK making `mapper_type` public, tho I might prefer a name like `mapper_hint`